app/javascript/styles/mastodon/components.scss

```diff
@@ -2963,6 +2963,15 @@ $ui-header-height: 55px;
   margin-bottom: 10px;
   display: flex;
   flex-direction: row;
+  border-radius: 4px;
+
+  &:first-child:hover {
+    border-radius: 4px 0 0 4px;
+  }
+
+  &:last-child:hover {
+    border-radius: 0 4px 4px 0;
+  }
```

Comment:
I wonder if it wouldn't make more sense for the `.drawer__header` to have `overflow: hidden`?

Reply:
Sure, you are right. I will change it soon.